frontend/feedbacks/views.py

```diff
@@ -1,14 +1,21 @@
 from django.http import HttpResponse, JsonResponse
 from django.shortcuts import render
-from .models import Evaluation_Feedback
+from .models import *
 import requests
 
 # Create your views here.
 def eval_feedback_view(request, *args, **kwargs):
     feedback = request.GET["feedback"]
-    eval_id = request.GET["eval_id"]
+    eval_block_id = request.GET["eval_block_id"]
 
     Evaluation_Feedback.objects.create(feedback=feedback, evaluation=eval_id)
```

Comment:
I dont know, where you getting eval_id, so i assume its should be eval_block_id
```suggestion
    Evaluation_Feedback.objects.create(feedback=feedback, evaluation=eval_block_id)
```

Reply:
You are correct, I forgot to change that. Thanks a lot.